### PR TITLE
dockerfile: rebase to f37 from f36

### PR DESCRIPTION
**Pull request by prestist**
https://github.com/coreos/fedora-coreos-tracker/issues/1223

### CODE REVIEW

**Review by dustymabe — APPROVED** [review]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.fedoraproject.org/fedora:36 AS builder
+FROM registry.fedoraproject.org/fedora:37 AS builder
```
Comment (jlebon) [inline Dockerfile:1]: This needs to update the second `FROM` below too.

**Review by jlebon — COMMENTED** [review]
We've previously noted these bumps in a "Packaging Changes" section. (And actually circling back to [this comment](https://github.com/coreos/ignition/pull/1505#issuecomment-1319247130), we should probably do that there too?)

File: Dockerfile (hunk 1 above)
Comment (prestist) [reply Dockerfile:1]: Awesome catch thank you, and ok sorry I was going off of the butane train, didnt look on all apps; assumed it was a standard not to mention it.

**Review by jlebon — APPROVED** [review]
Super minor/optional: for the commit message, `Dockerfile` would be more correct since that's the name of the file, and "rebase from f36 to f37" would read better. :)

**Pull request merged**